### DISCUSSION TITLE
Change Victoria for Orbit Runner

### DISCUSSION THREAD

**Discussion by Yunnnnnnnnnn** [Re-Volt-America/rva_pack#10] [RVA Cars]
A suggestion to change one of the current cars in the pack for another.

***Victoria*** got introduced this season (2022-2), and the few first sessions that it got used, the car was deemed too overpowered for the class, being able to compete against Pros and outclassing classics, such as *Dragoon*, the undisputed king of custom Semi-Pros.
Ghoster's ***Orbit Runner*** offers the same style of insane grip and stability in corners as Victoria, while having a very reduced top speed and RWD drivetrain problems, and being fairly easier to kill.

- The general debate of the players resides in either "leaving the car in the pack at 0.6" or completely removing it despite the car's uniqueness and appeal.
- The suggestion is changing it for a car that boasts the same style of play.
- Orbit Runner will be granted a multiplier of 0.7

TL;DR:
This -> [Victoria](https://www.revoltworld.net/dl/victoria/)
For this -> [Orbit Runner](https://www.revoltworld.net/dl/orbit-runner/)
